sippy-ng/src/component_readiness/ComponentReadiness.js

```diff
@@ -682,9 +679,6 @@ export default function ComponentReadiness(props) {
                     ignoreDisruption,
                     ignoreMissing
                   )
-                  setComponentParam(component)
-                  setCapabilityParam(capability)
-                  setEnvironmentParam(environment)
```

Comment:
I tested this and the one below and those lines are no longer necessary so this a good change.  It's interesting at the time, we needed these and how we don't.

I tested a similar set of setxxx lines for the `/component_readiness/env_test` route and those are redundant too.  I can look into removing those in a different PR.

Reply:
We needed them before because of the 1st issue in the description,  /component_readiness always takes you to the `<ComponentReadiness>` component, it doesn't get reloaded, so old state stays around.  You have to explicitly set the state to make it work.